yuhow/Makefile

```diff
@@ -1,76 +0,0 @@
-SHELL := bash
```

Comment:
EPS will resolve the pdflatex issue.  Moving back to latex+dvips allows us to use both EPS and PSTricks with PStake.  It is good.

A further step is how would we collaborating on drawings.  For mesh_and_ce_se_v1.pptx, I have https://github.com/solvcon/cesenote/blob/master/stcon/schematic/staggered_stencil.tex that you might find similar.  For sod_shock_tube_1D.pptx, I have something in my laptop that should have been shared.  They are both in PSTricks/LaTeX format, and can be converted to EPS by PStake.